hs_core/views/utils.py

```diff
@@ -1290,6 +1290,7 @@ def get_coverage_data_dict(source, coverage_type='spatial'):
         spatial_coverage_dict = {}
         if spatial_coverage:
             spatial_coverage_dict['type'] = spatial_coverage.type
+            spatial_coverage_dict['name'] = spatial_coverage.value['name']
```

Comment:
It is possible that 'name' may be missing in coverage data. So it needs to be:
`spatial_coverage_dict['name'] = spatial_coverage.value.get('name', "") `

Reply:
This fixed the test case. Thanks!